src/goRename.ts

```diff
@@ -22,17 +22,17 @@ export class GoRenameProvider implements vscode.RenameProvider {
 			var filename = this.canonicalizeForWindows(document.fileName);
 			var offset = document.offsetAt(position);
 
-			var gorename = path.join(process.env["GOPATH"], "bin", "gorename");
+			var gorename = getBinPath("gorename");
 
 			cp.execFile(gorename, ["-offset", filename + ":#" + offset, "-to", newName], {}, (err, stdout, stderr) => {
 				try {
 					if (err && (<any>err).code == "ENOENT") {
 						vscode.window.showInformationMessage("The 'gorename' command is not available.  Use 'go get golang.org/x/tools/cmd/gorename' to install.");
-						return resolve(null);
+						return Promise.resolve<vscode.WorkspaceEdit>(null);
```

Comment:
The original code caused that for me (since gorename isn't in my GOPATH/bin. It would would go into the block at line 29 and then return resolve(null).